src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Services/ContentApiClient.cs

```diff
@@ -57,6 +57,40 @@ public ContentApiClient(ILogger<ContentApiClient> logger, HttpClient httpClient)
             await response.Content.ReadFromJsonAsync<PaginatedListViewModel<PublicationSearchResultViewModel>>();
 
         return publications
-            ?? throw new Exception("Could deserialize publications from content API.");
+            ?? throw new Exception("Could not deserialize publications from content API.");
+    }
+
+    public async Task<Either<ActionResult, PublishedPublicationSummaryViewModel>> GetPublication(Guid publicationId)
+    {
+        var response = await _httpClient.GetAsync($"api/publications/{publicationId}/summary");
+
+        if (!response.IsSuccessStatusCode)
+        {
+            var message = await response.Content.ReadAsStringAsync();
+            _logger.LogError(StringExtensions.TrimIndent(
+                $"""
+                 Failed to retrieve publication '{publicationId}' with status code: {response.StatusCode}. Message:
+                 {message}
+                 """));
```

Comment:
In 400 or 404 scenarios, we probably don't need to log out an error as it's typically going to be user error that leads to this.

Consequently, we could just put this into the `default` case of the switch. It also saves on reading the response body and logging it out as well.

Probably should have spotted this earlier, but we're also doing this in `ListPublications`. Think we could do a similar thing there as well!

Reply:
Done :)